actions/qemuimg2disk/v1/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM alpine:3.13.5 as builder
+ENV QEMU_NAME="qemu"
+ARG QEMU_REV="6.0.0"
+ENV QEMU_SRC_BASENAME="${QEMU_NAME}-${QEMU_REV}"
+ENV QEMU_SRC_URL="https://download.qemu.org/${QEMU_SRC_BASENAME}.tar.xz"
+
+RUN apk update && \
+    apk upgrade && \
```

Comment:
still the same issue, being in the same docker layer wasn't the problem :p. You want `apk add --update --upgrade ...`